src/scss/themes/_standard.scss

```diff
@@ -1,7 +1,7 @@
 /// Inverse theme styles - base all text content on white
 @mixin oTeaserInverse {
-	.o-teaser__heading:hover,
-	.o-teaser__heading:focus,
+	.o-teaser__heading a:hover,
+	.o-teaser__heading a:focus,
```

Comment:
How do you mean? In other components or within o-teaser? If the latter then they should have focus states now

Reply:
@gvonkoss ah in the case of `.o-teaser__tag` etc, those selectors are for the link elements rather than containing divs like they are with the header.

Reply:
Good spot. Reverted!

Reply:
Wait. My mistake. I thought that was my change, and a different change. Need more morning coffee. You are right.

Reply:
Cool I've put your change back @benbarnett and updated the other instances I spotted. Sorry about that.

Reply:
Awesome, thanks @notlee!